templates/docs/examples/patterns/navigation/_script-sliding.js

```diff
@@ -255,8 +255,18 @@ const initNavigationSliding = () => {
     };
   };
 
-  // hide side navigation drawer when screen is resized
-  window.addEventListener('resize', throttle(closeAllDropdowns, 10));
+  // hide side navigation drawer when screen is resized horizontally
+  let previousWidth = window.innerWidth;
+  window.addEventListener(
+    'resize',
+    throttle(function () {
+      const currentWidth = window.innerWidth;
+      if (currentWidth !== previousWidth) {
```

Comment:
Could we add a comment here that this is ignoring vertical resize events?

Reply:
Do you not think the comment above 'hide side navigation drawer when screen is resized horizontally' is clear enough?

Reply:
It's clear enough for my purposes, just being a bit nit-picky